### ISSUE TITLE
Misnamed Columns

### ISSUE THREAD

**Issue by JalapenoGremlin** [philipmulcahy/azad#32]
```
Order Summary
Item(s) Subtotal: $39.90
Shipping & Handling: $0.00
Total before tax: $39.90
Estimated tax to be collected: $2.49
Grand Total: $42.39
```
This is the text from Amazon.com (US). 

The spreadsheet says VAT = $39.90; total=$42.39

But the Vat is (Value Added Tax, or Sales Tax in the US) is $2.49. The SUB-Total is $39.90

Thanks for creating this extension. It's got some real potential.

**Comment by philipmulcahy** [comment]
Thank you for the feedback! This is for dot com right? I'm guessing that the recent state tax ruling has caused this (with Amazon then adding a new line to their template). I think I know how to fix it. Won't have time until early next week though. Sorry...

On 6 July 2018 02:14:48 BST, Brian Carlson <notifications@github.com> wrote:
>```
>Order Summary
>Item(s) Subtotal: $39.90
>Shipping & Handling: $0.00
>Total before tax: $39.90
>Estimated tax to be collected: $2.49
>Grand Total: $42.39
>```
>This is the text from Amazon.com (US). 
>
>The spreadsheet says VAT = $39.90; total=$42.39
>
>But the Vat is (Value Added Tax, or Sales Tax in the US) is $2.49. The
>SUB-Total is $39.90
>
>Thanks for creating this extension. It's got some real potential. 
>
>
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/philipmulcahy/azad/issues/32

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by philipmulcahy** [comment]
I think I've fixed this with 1.8.11 (pushing out now).
Amazon changed the case of a single char: this is what I get/deserve for sharing a website scraping util.

**Comment by JalapenoGremlin** [comment]
Thanks, Philip!

Your work on delivering this scaping util is greatly appreciated. It's unfortunate that Amazon doesn't provide this. Your extension is the only way to get the Digital Order History.
